sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -76,33 +81,41 @@
                 )
               "
             >
-              <v-btn
-                :disabled="sinValidStatus !== SINStatusEnum.VALID"
-                variant="plain"
-              >
-                <font-awesome-icon
-                  :icon="['fas', 'pen']"
-                  class="mr-2"
-                  v-tooltip="'Click To Edit this Application'"
-                  @click="
-                    $emit(
-                      'editApplicationAction',
-                      slotProps.data.status,
-                      slotProps.data.id,
-                    )
-                  "
-                />
-              </v-btn>
-              <v-btn
-                :disabled="sinValidStatus !== SINStatusEnum.VALID"
-                variant="plain"
-              >
-                <font-awesome-icon
-                  :icon="['fas', 'trash']"
-                  v-tooltip="'Click To Cancel this Application'"
-                  @click="$emit('openConfirmCancel', slotProps.data.id)"
-                />
-              </v-btn>
+              <v-tooltip>
+                <template v-slot:activator="{ props }">
```

Comment:
Only for programmatic tool tip we need to wrap button inside <v-tooltip>. Otherwise e can define the the v-tooltip inside v-btn. Correct me if I am wrong.

Reply:
i didn't understand

Reply:
getting vue compilation error, 
![image](https://user-images.githubusercontent.com/77353155/182717791-80be63a1-fccf-4b68-b21d-9e17a7ce0f47.png)

Reply:
both methods are correct. updated to the one that use less code :)